Products/ZenCallHome/CallHomeStatus.py

```diff
@@ -110,9 +110,11 @@ def getStatUI(self):
     def _init(self):
         """Sets empty data for CallHomeStatus before run
         """
-        data = pickle.loads(self.load_from_redis())
-        if data == None:
+        rdata = self.load_from_redis()
+        if rdata == None:
```

Comment:
"Correct" Python requires this to be written as `if rdata is None:`.  Since `None` is a singleton value, you should use identity, i.e. `is`, rather than equality for comparison.